README.md

```diff
@@ -809,20 +620,367 @@ When you have time, we invite you to take a look at:
  + [The dynamic router EIP](https://camel.apache.org/components/latest/eips/dynamicRouter-eip.html)
  + [The kamelet EIP](https://camel.apache.org/components/latest/eips/kamelet-eip.html)
 
-## Part 7 - Camel Quarkus and Kafka
+## Part 5 - Camel Quarkus and Kafka
 Estimate time : 25 minutes
-@TODO: 25 MINUTES LEFT at this level
 
-## Bonus A - Camel Quarkus and the Kamelets
-@TODO
+### About this section
+In this part, we will discover how easy and fast it is to :
+* Stream data from and to Kafka
+* Write to Slack
+* Read / write to an SQL database with JPA
+* Use external APIs
+* Serialize Data
+* Create REST API
+
+
+We will also experience the joy of using [dev services](https://quarkus.io/guides/dev-services) in the development mode. No need to install any kafka server or database.
+
+### Introducing the example
+
+This example will illustrate an application that will receive and process coffee orders. The application is designed around Event Driven Architecture and uses Kafka to communicate with external systems. The orders are stored within a database. The delivery team is notified on slack to go check the order. The delivery team can access information about orders to deliver using an API.
+
+* insert the orders within a Database
+* notify the delivery company using a specific API (This one will be provided during the session)
+
+The application is designed around Event Driven Architecture and uses Kafka to handle those messages:
+* The coffee orders to be processed are in a kafka topic named `orders`
+* The notifications to the delivery company are in a kafka topic named `deliveries`
+
+The application that create orders and notify delivery teams are external to our ecosystem.
+
+![](resources/CQ-workshop1.png)
+
+Let's navigate to the part 5 folder.
+
+### Dev Services
+Quarkus supports the automatic provisioning of unconfigured services in development and test mode. For more information, you can check the [official Dev Services overview](https://quarkus.io/guides/dev-services).
+As we are using kafka and PostgreSQL, without any configuration, Quarkus dev services will start automatically Strimzi and PostgreSQL. 
+Start the app in dev mode. Notice that those 2 servers have been started.
```

Comment:
I see warnings here, not sure this is an issue:
```
2022-01-21 12:48:38,521 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread: <default>) SQL Warning Code: 0, SQLState: 00000

2022-01-21 12:48:38,522 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread: <default>) table "camel_messageprocessed" does not exist, skipping
2022-01-21 12:48:38,523 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread: <default>) SQL Warning Code: 0, SQLState: 00000
2022-01-21 12:48:38,523 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread: <default>) table "coffeeorder" does not exist, skipping
2022-01-21 12:48:38,523 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread: <default>) SQL Warning Code: 0, SQLState: 00000
2022-01-21 12:48:38,524 WARN  [org.hib.eng.jdb.spi.SqlExceptionHelper] (JPA Startup Thread: <default>) sequence "hibernate_sequence" does not exist, skipping
```
2022-01-21 12:48:39,038 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'sasl.kerberos.ticket.renew.window.factor' was supplied but isn't a known config.
2022-01-21 12:48:39,038 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'sasl.kerberos.kinit.cmd' was supplied but isn't a known config.
2022-01-21 12:48:39,039 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'specific.avro.reader' was supplied but isn't a known config.
2022-01-21 12:48:39,039 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'sasl.kerberos.ticket.renew.jitter' was supplied but isn't a known config.
2022-01-21 12:48:39,039 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'ssl.trustmanager.algorithm' was supplied but isn't a known config.
2022-01-21 12:48:39,039 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'ssl.keystore.type' was supplied but isn't a known config.
2022-01-21 12:48:39,039 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'sasl.kerberos.min.time.before.relogin' was supplied but isn't a known config.
2022-01-21 12:48:39,040 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'ssl.protocol' was supplied but isn't a known config.
2022-01-21 12:48:39,040 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'ssl.enabled.protocols' was supplied but isn't a known config.
2022-01-21 12:48:39,040 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'ssl.truststore.type' was supplied but isn't a known config.
2022-01-21 12:48:39,040 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) The configuration 'ssl.keymanager.algorithm' was supplied but isn't a known config.
2022-01-21 12:48:39,043 INFO  [org.apa.cam.com.kaf.con.sup.ResumeStrategyFactory] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) Using NO-OP resume strategy
2022-01-21 12:48:39,043 INFO  [org.apa.cam.com.kaf.KafkaFetchRecords] (Camel (camel-1) thread #1 - KafkaConsumer[orders]) Subscribing orders-Thread 0 to topic orders
2022-01-21 12:49:11,950 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:49:11,951 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:49:12,947 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:49:12,948 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:49:17,947 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:49:17,948 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:49:37,951 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:49:37,952 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:49:38,951 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:49:38,952 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:49:52,954 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:49:52,955 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:50:28,999 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:50:29,000 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:50:29,227 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {s}->https://random-data-api.com:443: The target server failed to respond
2022-01-21 12:50:29,228 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {s}->https://random-data-api.com:443
2022-01-21 12:50:41,068 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:50:41,069 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:50:41,295 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {s}->https://random-data-api.com:443: The target server failed to respond
2022-01-21 12:50:41,296 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {s}->https://random-data-api.com:443
2022-01-21 12:50:41,961 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:50:41,962 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:51:19,963 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:51:19,965 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:51:20,220 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {s}->https://random-data-api.com:443: The target server failed to respond
2022-01-21 12:51:20,221 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {s}->https://random-data-api.com:443
2022-01-21 12:51:34,965 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {}->http://random-data-api.com:80: The target server failed to respond
2022-01-21 12:51:34,966 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {}->http://random-data-api.com:80
2022-01-21 12:51:35,190 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {s}->https://random-data-api.com:443: The target server failed to respond
2022-01-21 12:51:35,192 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {s}->https://random-data-api.com:443
2022-01-21 12:52:12,174 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) I/O exception (org.apache.http.NoHttpResponseException) caught when processing request to {s}->https://random-data-api.com:443: The target server failed to respond
2022-01-21 12:52:12,175 INFO  [org.apa.htt.imp.exe.RetryExec] (Camel (camel-1) thread #2 - timer://create-random-coffee-orders) Retrying request to {s}->https://random-data-api.com:443

```

```

Reply:
Can you try again? The API is accessible right now for me